install_config/install/prerequisites.adoc

```diff
@@ -223,8 +277,63 @@ domain is not listed in the search list. Otherwise, containers managed by
 OpenShift may fail to resolve host names properly.
 ====
 
-[[prereq-network-access]]
 
+[[dns-config-prereq]]
+*Configuring a DNS Environment*
+
+To properly configure your DNS environment for {product-title}: 
+
+. Ensure that the DNS servers listed in *_/etc/resolv.conf_* are able to resolve the addresses of all the masters and nodes in your OpenShift environment:
++
+----
+$ cat /etc/resolv.conf
+$ dig <node_hostname> @<IP_address> +short
+----
++
+For example:
++
+----
+$ cat /etc/resolv.conf
+search example.com
+nameserver 172.16.25.35
+
+$ dig ose3-node1.example.com  @172.16.25.41 +short
+172.16.25.45
+----
++
+. Ensure that the local DNS servers listed in *_/etc/resolv.conf_* hosting your DNS records are listed first:
```

Comment:
In 3.2 we now will list the IP address of the host in the /etc/resolv.conf. This is done by a[script added to host](https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_node_dnsmasq/files/networkmanager/99-origin-dns.sh#L54) by the installer.